### DISCUSSION TITLE
Configuration discovery

### DISCUSSION THREAD

**Discussion by elupus** [yozik04/nibe#60] [Ideas]
I was thinking of how to simplify configuration of the client for a nibegw. A few options arises.

1. Custom frame on unused acessory id containing the read/write and future virtual rmu ports.
2. Mdns announce
3. Ssdp announce

I think 1 is easiest to implement. That way you only need to configure the listening port on you system and everything else can be discovered.

Mdns and ssdp would make the gw unit more complex,.but it would be a separation of concern and a somewhat more standard solution.

**Comment by yozik04** [comment]
I agree that it would be great to make an autodiscovery. I think mdns won't fit into Arduino nano flash.
